src/contexts/registration/RegistrationContext.js

```diff
@@ -0,0 +1,52 @@
+import React, { createContext, useState } from 'react';
+
+const RegistrationContext = createContext();
+
+export const RegistrationState = props => {
+  const [user, setUser] = useState({
+    email: '',
+    password: '',
+    passwordRepeat: ''
+  });
+
+  const [tooltip, setTooltip] = useState({
+    emailTooltip: false,
+    passwordTooltip: false,
+    passwordRepeatTooltip: false
+  });
```

Comment:
Nie wiem czy jest sens to wpychać w context, jeśli jest używane tylko w jednym komponencie (z innymi takimi state które nie wymagają komunikacji pomiędzy komponentami, może lepiej jest machnąć zwykłego state w komponencie)